examples/apps/generic_client/generic_client.py

```diff
@@ -289,66 +436,37 @@ def Main(args=None):
             WorkOrderStatus.PENDING:
         sys.exit(1)
 
-    wo_receipt_instance = direct_jrpc.get_work_order_receipt_instance()
     # Create receipt
+    wo_receipt = WorkOrderReceiptJRPCClientImpl(config)
     if show_receipt:
-        req_id += 1
-        # Create work order receipt object using WorkOrderReceiptRequest class
-        wo_request = json.loads(wo_params.to_jrpc_string(req_id))
-        wo_receipt_obj = WorkOrderReceiptRequest()
-        wo_create_receipt = wo_receipt_obj.create_receipt(
-            wo_request,
-            ReceiptCreateStatus.PENDING.value,
-            private_key
-        )
-        logger.info("Work order create receipt request : {} \n \n ".format(
-            json.dumps(wo_create_receipt, indent=4)
-        ))
-        # Submit work order create receipt jrpc request
-        wo_receipt_resp = wo_receipt_instance.work_order_receipt_create(
-            wo_create_receipt["workOrderId"],
-            wo_create_receipt["workerServiceId"],
-            wo_create_receipt["workerId"],
-            wo_create_receipt["requesterId"],
-            wo_create_receipt["receiptCreateStatus"],
-            wo_create_receipt["workOrderRequestHash"],
-            wo_create_receipt["requesterGeneratedNonce"],
-            wo_create_receipt["requesterSignature"],
-            wo_create_receipt["signatureRules"],
-            wo_create_receipt["receiptVerificationKey"],
-            req_id
-        )
-        logger.info("Work order create receipt response : {} \n \n ".format(
-            wo_receipt_resp
-        ))
+        jrpc_req_id += 1
+        _create_work_order_receipt(wo_receipt, wo_params,
```

Comment:
_create_work_order_receipt() is doing creation of receipt and submitting request but _create_work_order() is creating a only request. Either move work order submission part also to _create_work_order() or change other APIs to do the same.

Reply:
will change _create_work_order -> _create_work_order_param. _create_work_order_receipt internally calls " work_order_receipt_create" function in https://github.com/hyperledger/avalon/blob/master/client_sdk/avalon_client_sdk/direct/jrpc/work_order_receipt_jrpc_client.py#L36. So to maintain consistency used this function name. We can add a comment in the function that this will send Work Order Receipt json rpc request